### ISSUE TITLE
File logging stops after adding db scheduler dependency in pom.xml

### ISSUE THREAD

**Issue by amit-onappr** [kagkarlsson/db-scheduler#277]
My project is having logback-spring.xml which has the log configuration. This works fine without havingn db-scheduler. Let me know if someone else has faced similar issue.

**Comment by kagkarlsson** [comment]
I cannot see that this should be caused by db-scheduler. Did you add more dependencies than db-scheduler?

**Comment by amit-onappr** [comment]
No, I didn't add other dependencies.  However when I remove db-scheduler dependency, file log starts coming else only console log.

**Comment by kagkarlsson** [comment]
Ok. I think you need to debug what is happening. You might try a `mvn dependency:tree` to see if something is adding a slf4j implementation other than yours

**Comment by amit-onappr** [comment]
Yeah. I am debugging this one.. I'll update once I find the solution. Thanks for your prompt reply.

**Comment by amit-onappr** [comment]
so if I rename my file to logback-spring.xml to logback.xml it works.
